src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/BedAssignmentHistoriesRepository.java

```diff
@@ -3,17 +3,21 @@
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.Pageable;
 import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
 import org.springframework.data.repository.PagingAndSortingRepository;
 import org.springframework.data.repository.query.Param;
 import uk.gov.justice.hmpps.prison.repository.jpa.model.BedAssignmentHistory;
+import uk.gov.justice.hmpps.prison.repository.jpa.model.BedAssignmentHistory.BedAssignmentHistoryPK;
 
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.util.List;
 import java.util.Optional;
 import java.util.Set;
 
-public interface BedAssignmentHistoriesRepository extends PagingAndSortingRepository<BedAssignmentHistory, BedAssignmentHistory.BedAssignmentHistoryPK> {
+public interface BedAssignmentHistoriesRepository extends
```

Comment:
CrudRepository isn't now automatically included if PagingAndSortingRepository specified, so needs to be included manually